docker-compose.prod.yml

```diff
@@ -0,0 +1,8 @@
+---
+version: '3'
+services:
+  freqtrade:
+    image: freqtradeorg/freqtrade:master
+    volumes:
```

Comment:
i guess we could add a (commented out) section about `restart: unless-stopped`?

if running the bot in production, i may want to restart it automatically (i do that with dry-run instances anyway). wouldn't make it default though, as users may have different opinions about that.

Reply:
I'm not sure what you want here.  I want to be able to run this locally with ease, that's what docker-compose offers me as a developer.  I want to contribute to this fork but in order to do so I need to be able to run locally.

Reply:
![image](https://user-images.githubusercontent.com/419355/65456322-47f63a80-de06-11e9-90c6-3c74d1bad0b8.png)